.php_cs

```diff
@@ -1,6 +1,6 @@
 <?php
 
-require_once './vendor/autoload.php';
+require __DIR__.'/vendor/sllh/php-cs-fixer-styleci-bridge/autoload.php';
```

Comment:
You might have some common library conflict with the composer autoload.

See: https://github.com/Soullivaneuh/php-cs-fixer-styleci-bridge#troubleshooting